app/code/Magento/PageBuilder/Controller/Adminhtml/ContentType/Block/Metadata.php

```diff
@@ -42,9 +42,14 @@ public function execute()
         try {
             $collection = $this->blockCollectionFactory->create();
             $blocks = $collection
-                ->addFieldToSelect(['title','is_active'])
-                ->addFieldToFilter('block_id', ['eq' => $params['block_id']])
-                ->load();
+                ->addFieldToSelect(['title', 'is_active'])
+                ->addFieldToFilter(
+                    ['block_id', 'identifier'],
+                    [
+                        ['eq' => $params['block_id']],
+                        ['eq' => $params['block_id']]
```

Comment:
It seems to me a typo, please review this condition `['eq' => $params['block_id']]` is repeating.